server.go

```diff
@@ -68,6 +70,11 @@ var (
 			"signrpc", "walletrpc", "chainrpc", "invoicesrpc",
 		},
 	}
+
+	// defaultClientPingTime is the default time we'll use for the client
+	// keepalive ping time. This means the client will ping the server every
+	// 10 seconds to make sure the TCP connection is still alive.
```

Comment:
nit: I don't think it will ping the server every 10 seconds. I think it will ping if it hasn't seen any activity after 10 seconds and then it will wait 20 seconds for a response.
```
	// After having pinged for keepalive check, the client waits for a duration
	// of Timeout and if no activity is seen even after that the connection is
	// closed.
	Timeout time.Duration // The current default value is 20 seconds.
```

from `google.golang.org/grpc@v1.39.0/keepalive/keepalive.go`

We might want to set `PermitWithoutStream` to `true` also so that pings occur even without active RPC.

Reply:
You're right, it's only when there is no other activity. Updated the comment.

I don't think we want the keepalive without a stream. Because at some point the number of open TCP connections will become a limiting factor for the number of Pool clients we can support. So we don't want to allow the client to keep a TCP connection open for longer than necessary. So if they don't have an account and therefore no account subscription stream, they shouldn't be able to keep open the connection artificially.